variables.tf

```
@@ -111,3 +100,30 @@ variable "host_name" {
   default     = "bastion"
   description = "The Bastion hostname created in Route53"
 }
+
+variable "user_data_template" {
+  type        = string
+  default     = "userdata/amazon-linux.sh"
```

Let's use `user_data` for consistency
```suggestion
  default     = "user_data/amazon-linux.sh"
```

---

Updated